style.css

```diff
@@ -369,7 +375,37 @@ footer {
   display: none;
 }
 
-@media (min-width: 600px) {
+.menuItems {
+  display: none;
+  background-color: #fff;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 375px;
+  height: 772px;
+  list-style-type: none;
+  padding-top: 20px;
+}
+
+.listCan {
+  padding: 30px 0;
+}
```

Comment:
- [ ] This is close but kindly make sure to stick to the Figma design as I noticed the space between the menu options is a bit tight I'd suggest you add more spacing so you can achieve a more polished UI.
- Figma design:
  ![image](https://github.com/Wils-FOURR-speed/My-portfolio/assets/104968236/fc627e81-9f27-4c48-bbea-01399fc22405)
- Your design:
  ![image](https://github.com/Wils-FOURR-speed/My-portfolio/assets/104968236/eb934342-186c-4977-813e-66c1a2c68938)